tests/tensorflow/nightly/common.libsonnet

```diff
@@ -129,11 +213,7 @@ local mixins = import 'templates/mixins.libsonnet';
   },
   local functional_schedule = '0 9 * * 3',
   Functional:: mixins.Functional {
-    schedule:
-      if !(self.accelerator.type == 'tpu') || self.accelerator.name == 'v3-8' || self.accelerator.name == 'v4-8' then
-        functional_schedule
-      else
-        null,
+    schedule: functional_schedule,
```

Comment:
This will re-enable a number of currently disabled tests that currently have their schedules set to `null` here, e.g. `tf.nightly-resnet-imagenet-func-v2-8.yaml`

I'm fine removing this logic (it's really not very intuitive), but please either remove the currently-disabled test configs or mark them with `+ mixins.Experimental` (this will set the schedule to `null`).

Reply:
I think its better to leave this as is.